lib/org/chromium/webidl/ParserRunner.js

```diff
@@ -8,45 +8,78 @@ foam.CLASS({
   name: 'ParserRunner',
   extends: 'org.chromium.webidl.PipelineRunner',
 
-  requires: ['org.chromium.webidl.BaseParser'],
+  requires: [
+    'org.chromium.webidl.BaseIDLFile',
+    'org.chromium.webidl.BaseParser',
+    'org.chromium.webidl.ast.Definition',
+    'org.chromium.webidl.Parser as DefaultParser',
+  ],
 
   documentation: 'Runnable Box that parsers IDLFileContents into ASTs.',
 
-  methods: [
-    function run(message) {
-      if (this.validateMessage(message)) return;
+  properties: [
+    {
+      class: 'Class',
+      documentation: 'Class of the parser used to parse the IDL files.',
+      name: 'Parser',
```

Comment:
Since this is a property, please use `camelCase`. You can call it `parserCls` or similar to clarify that it's a class object.